inc/computer.class.php

```diff
@@ -67,7 +67,12 @@ function rawSearchOptions() {
       $plugin = new Plugin();
       if ($plugin->isInstalled('fields')) {
          if ($plugin->isActivated('fields')) {
-            include_once(GLPI_ROOT . "/plugins/fields/hook.php");
+            // Check if Fields plugin is installed in marketplace or plugins folder
```

Comment:
The `Plugin::getPhpDir()` method could be used here.

Reply:
Didn't knew this function, I've made the change

Reply:
No problem ;) It's been introduced in 9.5 especially for that ;)